website/screens/components/wizard/code/examples/uncontrolled.ts

```diff
@@ -1,44 +1,30 @@
 import { DxcWizard, DxcInset } from "@dxc-technology/halstack-react";
 
 const code = `() => {
-    const onStepClick = i => {
-      console.log(i);
-    };
+  const onStepClick = (i) => {
+    console.log(i);
+  };
 
-    return (
-      <DxcInset space="large">
-        <DxcWizard
-          defaultCurrentStep={1}
-          onStepClick={onStepClick}
-          steps={[
-            {
-              label: "Cart"
-            },
-            {
-              label: "Address",
-              icon: (
-                <svg
-                      viewBox="0 0 24 24"
-                      enable-background="new 0 0 24 24"
-                      fill="currentColor"
-                    >
-                  <g id="Bounding_Box">
-                    <rect fill="none" width="24" height="24" />
-                  </g>
-                  <g id="Master">
-                    <path d="M19,9.3V4h-3v2.6L12,3L2,12h3v8h5v-6h4v6h5v-8h3L19,9.3z M10,10c0-1.1,0.9-2,2-2s2,0.9,2,2H10z" />
-                  </g>
-                </svg>
-              )
-            },
-            {
-              label: "Payment"
-            },
-         ]}
-        ></DxcWizard>
-      </DxcInset>
-    );
-  }`;
+  return (
+    <DxcInset space="large">
+      <DxcWizard
+        defaultCurrentStep={1}
```

Comment:
We could remove `defaultCurrentStep` to make the code cleaner.

Reply:
I think it could be a good idea to add `defaultValue` to all uncontrolled examples. It would show how it works.

Reply:
Yes, but why only that prop and not any other? I mean, it is just a prop that doesn't really have any difficulty to add, not like in in the case of onChange or onBlur that needs to create an additional function.

Reply:
`defaultValue` prop is used only for uncontrolled components, for giving them a default value without being a controlled one. That's why it is useful in an uncontrolled example.

Reply:
Fair enough, you are right.